structures/insertion_sort.py

```diff
@@ -0,0 +1,53 @@
+# -*- coding utf-8 -*-
+from __future__ import unicode_literals
+
+
+def insertion_sort(iterable):
+    """Return a sorted iterable.
+    Takes an unsorted iterable and sorts it using the insertion method.
+    """
+    for i in xrange(1, len(iterable)):
+        item = iterable[i]
+        j = i
+        while item < iterable[j - 1] and j > 0:
+            iterable[j] = iterable[j - 1]
+            j -= 1
+        iterable[j] = item
+    return iterable
+
+
+if __name__ == '__main__':
+    from timeit import timeit
+    from random import randint
+
+    num = 10 ** 4
+
+    best = [x for x in xrange(num)]
+    worst = [x for x in xrange(num, -1, -1)]
+    average = [randint(0, num) for x in xrange(num)]
+
+    setup = """
+from __main__ import insertion_sort, num, best, worst, average
+
+"""
```

Comment:
This looks a bit strange from a readability standpoint because it breaks your indentations. What about this to keep your indentations consistent:

```python
    setup = (
        "from __main__ import insertion_sort, num, best, worst, average"
    )
```

Either that or just go ahead and break the 80 char limit rule. The [Google Python Style Guide](https://google-styleguide.googlecode.com/svn/trunk/pyguide.html?showone=Line_length#Line_length) excepts long imports from the rule.